common/scripted_effects/cwp_political_setup.txt

```diff
@@ -45,26 +47,18 @@ setup_starting_ideologies = {
 			remove_ideology = ideology_socialist
 			add_ideology = ideology_social
 		}
-		if = {
-			limit = {
-				owner = {
-					or = {
-						has_law = law_type:law_consensus_democracy
-						has_law = law_type:law_direct_democracy
-					}
-				}
-			}
-			remove_ideology = ideology_vanguardist
-			remove_ideology = ideology_social
-			remove_ideology = ideology_socialist
-			add_ideology = ideology_revolutionary
-		}
 		if = {
 			limit = {
 				owner = {
 					or = {
 						has_law = law_type:law_anocracy
 						has_law = law_type:law_oligarchy
+						has_law = law_type:law_autocracy
+					}
+					and = {
+						has_law = law_type:law_socialization
+						has_law = law_type:law_total_nationalization
+						has_law = law_type:law_heavy_industry_nationalization
```

Comment:
This `and` block says the nation must have both `total_nationalization` **and** `heavy_industry_nationalization` which is impossible.